### PR TITLE
feat: enable usage analytics reporting from VS Code

### PR DESCRIPTION
**Pull request by NetaMolcho**

### CODE REVIEW

**Comment by NetaMolcho** [comment]
image

File: README.md [hunk 1]
```diff
@@ -63,3 +63,6 @@ To get more help, support, and information please open a github [issue](https://
 
 ## Contributing
 Contributing information can be found in the [CONTRIBUTING.md](CONTRIBUTING.md) file.
+
+## Enable usage analytics reporting from VS Code
+The tool collects non-personally identifiable information about your usage of the tool to improve its services. If you do not want the tool to collect your usage data, you can set the "Enable Sap Web Analytics" setting to "false". Go to File > Preferences > Settings (macOS: Code > Preferences > Settings) > Extensions > <Tool Name>, and deselect the "Enable Sap Web Analytics" checkbox.
```
Comment (tomer-epstein) [inline README.md:68]: place this paragraph after--> #### Run the VSCode extension

**Review by tomer-epstein — CHANGES_REQUESTED** [review]

**Review by tomer-epstein — DISMISSED** [review]

**Comment by tomer-epstein** [comment]
Tested both in VS and AppStudio?

File: backend/package.json [hunk 2]
```diff
@@ -83,6 +83,12 @@
 			"type": "object",
 			"title": "Application Wizard",
 			"properties": {
+				"ApplicationWizard.enableSapWebAnalytics": {
+					"type": "boolean",
+					"default": true,
```
Comment (tomer-epstein) [inline backend/package.json:88]: Can BAS developers choose not to report WebAnalytics? (change the default to false)

**Review by tomer-epstein — APPROVED** [review]

**Pull request merged**